src/arsenic/session.py

```diff
@@ -315,6 +315,10 @@ async def switch_to_window(self, handle):
             url="/window", method="POST", data={"handle": handle, "name": handle}
         )
 
+    async def new_window(self, window_type: WindowType = None):
```

Comment:
Please update `docs/reference/session.rst`

Reply:
Thanks for a tip. I have updated session.rst. Please have a look.